backend_plugin/src/activation/Startup.java

```diff
@@ -19,7 +18,8 @@ public void run() {
 				if (window != null) {
 					// Once we are loaded, make a new FeatureSuggestion
 					// Need to determine how FE will get the FeatureSuggestion object
-					new FeatureSuggestion();
+					FeatureSuggestion fs = new FeatureSuggestion();
```

Comment:
If we aren't doing anything between construction and calling `start()`, then it's might worth it make the constructor start the service as well. I know this file will go through a lot of change as we're figuring out the interface with the front end, so it's probably fine for now, but something to think about.